components/brave_wallet/browser/swap_response_parser.cc

```diff
@@ -388,9 +390,12 @@ mojom::BlockchainTokenPtr ParseToken(const swap_responses::LiFiToken& value) {
   auto result = mojom::BlockchainToken::New();
   result->name = value.name;
   result->symbol = value.symbol;
-  result->logo = value.logo_uri;
+  result->logo = value.logo_uri.has_value() ? value.logo_uri.value() : "";
   result->contract_address =
-      value.address == kLiFiNativeAssetContractAddress ? "" : value.address;
+      value.address == kLiFiNativeEVMAssetContractAddress ||
+              value.address == kLiFiNativeSVMAssetContractAddress
```

Comment:
I'd wrap it like this for clarity:
`(value.address == kLiFiNativeEVMAssetContractAddress || value.address == kLiFiNativeSVMAssetContractAddress)`

Reply:
wrapped